### PR TITLE
ci: modify platform deployment method

### PR DESCRIPTION
**Pull request by mzeevi**
This PR introduces changes to the way the platform is deployed, especially its prereq, which now use Helm charts to deploy Capp. This makes it possible to fully test the platform backend with a functioning DNS server available.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,48 +18,125 @@ The API is documented in the [docs/api](./docs/api) directory of this repository
...
 ```bash
 $ make run
 ```
 
-Alternatively, to build the backend as a Docker image, use the following `Makefile` targets:
+### Deploy to OpenShift
+
+#### Prereq
+
+To install all the prerequisites to run the `platform-backend` on an OpenShift cluster, use the following `Makefile` target, which configures the `Capp` environment to work properly using an actual DNS Server.
```
Comment (fran2810) [inline README.md:70]: can you address the minimal amount of value you need for the make prereq to work ?

File: hack/convert-yaml-to-env.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
```
Comment (fran2810) [inline hack/convert-yaml-to-env.sh:2]: i dont get why we need it

Comment (mzeevi) [reply hack/convert-yaml-to-env.sh:2]: We don't, it's from one of the iterations, I'll remove it

File: README.md (hunk 1 above)
Comment (mzeevi) [reply README.md:70]: For a fully functioning environment you need all the values listed in the table below.

**Review by fran2810 — APPROVED** [review]
/lgtm

**Pull request merged**